packages/vite/src/node/build.ts

```diff
@@ -229,16 +236,60 @@ export interface LibraryOptions {
 
 export type LibraryFormats = 'es' | 'cjs' | 'umd' | 'iife'
 
-export type ResolvedBuildOptions = Required<BuildOptions>
+export interface ModulePreloadOptions {
+  /**
+   * whether to inject module preload polyfill.
+   * Note: does not apply to library mode.
+   * @default true
+   */
+  polyfill?: boolean
+  /**
+   * Resolve the list of dependencies to preload for
+   * a given dynamic import
```

Comment:
I don't know if there's a reason to use a short line length, but this looks like it could probably fit on one line
```suggestion
   * Resolve the list of dependencies to preload for a given dynamic import
```